pom.xml

```diff
@@ -1631,6 +1632,11 @@
         <artifactId>httpcore</artifactId>
         <version>${httpcore.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
```

Comment:
Why not also include `httpcore` (as in #563)?

Reply:
httpcore was already included in the pom.xml for this branch that's why I added httpclient only.